src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -218,6 +192,36 @@ export default class DetailsBody extends React.Component<Props> {
                 <Placeholder height="200px" />
               )}
             </ChartPanel>
+            <ChartActions>
```

Comment:
Can you pull this into a function called `renderChartActions`, just to keep things relatively readable :)